README.md

```diff
@@ -35,7 +35,12 @@ package in 'developer mode' using `pip`, in a virtual environment. You also need
 ```sh
 $ pip install -e .
 ```
+## Prerequisites
+You'll need:
+- pandoc: http://pandoc.org/installing.html
```

Comment:
Actually, we don't really need `pandoc` anymore. So we can probably remove it altogether from the README + setup.py. `phantomjs` should be enough currently.